### PR TITLE
Editorial: modernize the language around actions

### PR DESCRIPTION
**Pull request by annevk**
This will make it easier to give them more associated concepts.

***
<a href="https://whatpr.org/notifications/212.html" title="Last updated on Jul 10, 2024, 12:20 PM UTC (77e7817)">Preview</a> | <a href="https://whatpr.org/notifications/212/96a2c28...77e7817.html" title="Last updated on Jul 10, 2024, 12:20 PM UTC (77e7817)">Diff</a>

### CODE REVIEW

**Comment by annevk** [comment]
I went ahead with this as it's editorial. Happy to address any nits of course, as always.

File: notifications.bs [hunk 1]
```diff
@@ -922,11 +935,10 @@ not.onclick = function() { displaySong(this); };</pre>
...
 <code>notificationclick</code> events on the {{ServiceWorkerGlobalScope}}.
 
-<p>Here a service worker shows a notification with a single "Archive"
-<a lt="actions">action</a>, allowing users to perform this common task from the
-notification without having to open the website (for example the notification
-platform might show a button on the notification). The user can also activate
-the main body of the notification to open their inbox.
+<p>Here a service worker shows a notification with a single "Archive" <a for=/>action</a>, allowing
+users to perform this common task from the notification without having to open the website (for
+example the, notification platform might show a button on the notification). The user can also
```
Comment (saschanaz) [inline notifications.bs:940]:
```suggestion
example, the notification platform might show a button on the notification). The user can also
```

**Review by saschanaz — COMMENTED** [review]
The only nit, otherwise looks fine.

File: notifications.bs (hunk 1 above)
Comment (annevk) [reply notifications.bs:940]: Oh wow, how did that happen. Thanks!

**Pull request merged**